webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -44,6 +48,8 @@ interface Callbacks {
   onDisplayStream?: (stream: MediaStream, label: string) => void;
   onDisplayTrack?: (ctx: TrackContext) => void;
   onHideTrack?: (ctx: TrackContext) => void;
+  onToggleVideo?: (streamId: string) => void;
+  onToggleAudio?: (streamId: string) => void;
```

Comment:
This can be missleading as it can indicate that **we** should toggle video or audio. Maybe `onParticipantToggleAudio` and `onParticipantToggleVideo`

In the future we would have
```ts
onParticipantToggleAudio?: (participantId: string) => void;
onParticipantToggleVideo?: (participantId: string) => void;
```